client/scripts/components/Modals/TorrentDetailsModal/TorrentGeneralInfo.js

```diff
@@ -82,7 +82,7 @@ class TorrentGeneralInfo extends React.Component {
         <li className="torrent-details__detail torrent-details__detail--tags">
           <span className="torrent-details__detail__label">Tags</span>
           <span className="torrent-details__detail__value">
-            {(torrent.tags.length == 0 ? this.getTags(torrent.tags) : <ValueNotAvailable />)}
+            {(torrent.tags.length ? this.getTags(torrent.tags) : <ValueNotAvailable />)}
```

Comment:
`torrent.tags.length` always returns 1 because `"".split(",").length` is 1. What do you think is the smartest way to deal with this?